main.go

```diff
@@ -94,51 +120,76 @@ func getJoke(w http.ResponseWriter, r *http.Request) {
 				result = append(result, item)
 			}
 		}
-		t.ExecuteTemplate(w, "findjoke", result)
+		err := t.ExecuteTemplate(w, "findjoke", result)
+		if err != nil {
+			log.Fatal(err)
+		}
```

Comment:
I believe in the situation when client sends two parameters (text and id), we will have execution of template 2 times. Maybe it's better to out a return statement here to prevent it?